src/video_core/renderer_vulkan/renderer_vulkan.cpp

```diff
@@ -895,6 +895,19 @@ void RendererVulkan::SwapBuffers() {
         secondary_window->PollEvents();
     }
 #endif
+
+#ifdef ANDROID
+    if (secondary_window) {
+        const auto& secondary_layout = secondary_window->GetFramebufferLayout();
+        if (!second_window) {
```

Comment:
I know this isn't your fault, but the presence of a variable called both `secondary_window` and `second_window` in the same scope which have different values is really dumb.

We should probably address that here while we're touching this code.

Reply:
> my limited pushback here is that the two equivalent variables for the primary window are "window" and "main_window" which are also pretty confusable! If I go to change one, should I also change the other?

Yeah it's probably a good idea to change both